### ISSUE TITLE
Missing tags for 2.0.4 and 2.0.5

### ISSUE THREAD

**Issue by GregOriol** [killbill/killbill-admin-ui#270]
It seems that docker images are available for version 2.0.4 and 2.0.5 but there is not tag corresponding in the repository.

It would be practical to have those tags to be able to compare released versions.

**Comment by pierre** [comment]
Wrong repo :-)

This is the mountable engine gem, used by the main Kaui application (https://github.com/killbill/killbill-admin-ui-standalone/commit/9f82e33c0c7fc0d40d066048723284d57c119224).

I'm fairly certain we have a bug tracking the need to tag the main repo during the release process, but I couldn't find it...

**Comment by GregOriol** [comment]
Isn't the ddl in this repository?

I guess tags should be made in both repositories.

**Comment by pierre** [comment]
Just to be clear:

* This is the _main_ repo, i.e. this is the core gem with most of the Kaui screens (and where the DDL is). Tags are created here as part of the release process.
* Some companies [embed](https://guides.rubyonrails.org/engines.html) this Kaui gem in their own application. Others (and our Docker images) use the [standalone application](https://github.com/killbill/killbill-admin-ui) instead.
* In the standalone repo, there is no easy way to see which version of the gems it uses (the source of truth for the Docker images would be on our private build machine). This is because the `Gemfile.lock` isn't checked-in anymore: we removed it as the app could be started on pure Ruby or on JRuby.
* What is probably really confusing is that the tags are so close to one another (the current version of the standalone app is 2.0.5, which uses the Kaui gem 2.0.3).


tl;dr: it's not great and we should do better, but this hasn't been a priority. Typically, when we release the Kaui gem, we always release a new version of the standalone app, so you can always assume they go hand in hand (and 90% of the changes you'd care about would be in this repo).

**Comment by pierre** [comment]
The standalone repo is now tagged as part of our release process.
